packages/jest-cli/src/cli/index.ts

```diff
@@ -48,17 +41,20 @@ export async function run(maybeArgv?: Argv, project?: Path) {
   }
 }
 
-export const buildArgv = (maybeArgv: ?Argv) => {
+export const buildArgv = (maybeArgv?: Array<string>): Config.Argv => {
   const version =
-    VERSION + (__dirname.includes(`packages${path.sep}jest-cli`) ? '-dev' : '');
+    getVersion() +
+    (__dirname.includes(`packages${path.sep}jest-cli`) ? '-dev' : '');
 
-  const rawArgv: Argv | string[] = maybeArgv || process.argv.slice(2);
-  const argv: Argv = yargs(rawArgv)
+  const rawArgv: Config.Argv | Array<string> =
+    maybeArgv || process.argv.slice(2);
+  const argv: Config.Argv = yargs(rawArgv)
     .usage(args.usage)
     .version(version)
     .alias('help', 'h')
     .options(args.options)
     .epilogue(args.docs)
+    // @ts-ignore: it's unable to infer what arguments it contains
```

Comment:
@jeysal could you maybe help out here?

Reply:
https://github.com/jeysal/jest/commit/07d0b41aa72f095307f0b99a8758a05163a15e0e

Reply:
amazing, thanks again!